ngshare/ngshare.py

```diff
@@ -471,7 +520,48 @@ def post(self, course_id):
         'Add or update students. (instructors only)'
         course = self.find_course(course_id)
         self.check_course_instructor(course)
-
+        students_str = self.get_argument('students', None)
+        # Check request format
+        if not students_str:
+            self.json_error(400, 'Please supply students')
+        try:
+            students = json.loads(students_str)
+        except json.decoder.JSONDecodeError:
+            self.json_error(400, 'Students cannot be JSON decoded')
+        if not isinstance(students, list):
+            self.json_error(400, 'Incorrect request format')
+        if not students:
```

Comment:
Redundant check? Not that it matters that much.

Reply:
Which line are you referring to? I see 4 lines in Github. 
Possible rejected requests:
```
"some string can't be JSON decoded
{"I": "am passing a dict instead"}
[]     # empty list
["username": ["rm", "-rf", "genguazaidian"]]
```